CRM/Contact/BAO/Group.php

```diff
@@ -447,28 +447,27 @@ public static function create(&$params) {
         $params['parents'] = [$domainGroupID];
       }
 
-      // first deal with removed parents
-      if (array_key_exists('parents', $params) && !empty($currentParentGroupIDs)) {
-        foreach ($currentParentGroupIDs as $parentGroupId) {
-          // no more parents or not in the new list, let's remove
-          if (empty($params['parents']) || !in_array($parentGroupId, $params['parents'])) {
-            CRM_Contact_BAO_GroupNesting::remove($parentGroupId, $params['id']);
+      if (!CRM_Utils_System::isNull($params['parents'])) {
```

Comment:
Hmm, `CRM_Utils_System::isNull` is somewhat confusing. Empty array is considered Null so this doesn't work.